packages/admin-panel-server/src/viz-builder/dashboardVisualisation/DashboardVisualisationExtractor.ts

```diff
@@ -48,21 +48,13 @@ export class DashboardVisualisationExtractor<
   };
 
   private vizToDashboardItem() {
-    const { code, name, legacy } = this.visualisation;
```

Comment:
We used to extract the name from the dashboardItem.config and keep it as a top level field. Decided to scrap that as it was breaking the schema validation and it's a bit confusing anyway.